assets/i18n/active.pl.json

```diff
@@ -0,0 +1,137 @@
+        "other": "Nie"
+    },
+    "command.default.yes": {
+        "hash": "sha1-5397e0583f14f6c88de06b1ef28f460a1fb5b0ae",
+        "other": "Tak"
+    },
+    "command.error.generic": {
+        "hash": "sha1-73c841bf1e817ec2ab09b17eb0b43ec61bf1eda9",
+        "other": "Coś poszło nie tak. Spróbuj później."
+    },
+    "command.error.invalidInput": {
+        "hash": "sha1-6ef073c836a398485967d7c10a085b611d272ddc",
+        "other": "Nieprawidłowy wpis: {{.Error}}"
+    },
+    "command.error.invalidNumberOfOptions": {
+        "hash": "sha1-31287872531872ed11e9ae69975afc1cd2ee09ae",
+        "other": "Musisz podać co najmniej dwie odpowiedzi albo żadnej."
+    },
+    "command.help.text.options": {
+        "hash": "sha1-0b06ca6a7a2a1e2481ed761fe93eefec7d67079e",
+        "other": "Możesz dostowować opcje wpisując `/{{.Trigger}} \"Pytanie\" \"Odpowiedź 1\" \"Odpowiedź 2\" \"Odpowiedź 3\"`"
+    },
+    "command.help.text.pollSetting.anonymous": {
+        "hash": "sha1-fb98b7ba3c20091f6ea79a47ee43122d14e3f50a",
+        "other": "Nie pokazuj kto na co głosował"
+    },
+    "command.help.text.pollSetting.introduction": {
+        "hash": "sha1-5cf3ba00c35f9b75d8b2118081c7c45b2a36ecb3",
+        "other": "Dodatkowe ustawienia ankiety, np.: `/{{.Trigger}} \"Pytanie\" \"Odpowiedź 1\" \"Odpowiedź 2\" \"Odpowiedź 3\" --progress --anonymous`. Dostępne ustawienia sondy:"
+    },
+    "command.help.text.pollSetting.progress": {
+        "hash": "sha1-9b8deba20f6a2fc6221841de9db19c903d28f97f",
+        "other": "Podczas sondy, pokaż ile głosów uzyskała każda odpowiedź."
+    },
+    "command.help.text.pollSetting.public-add-option": {
+        "hash": "sha1-97f6e52cd7135ca4c73b79790a946ead7513cd33",
+        "other": "Zezwalaj użytkownikom na dodanie własnych odpowiedzi."
+    },
+    "command.help.text.simple": {
+        "hash": "sha1-783c9e50cbbd9a11685f41524265201b2e97d369",
+        "other": "Żeby utworzyć sondę z odpowiedziami \"{{.Yes}}\" i \"{{.No}}\" wpisz `/{{.Trigger}} \"Pytanie\"`"
+    },
+    "dialog.addOption.element.displayName": {
+        "hash": "sha1-e31d972229de381b461860014dd710856361917a",
+        "other": "Opcja"
+    },
+    "dialog.addOption.submitLabel": {
+        "hash": "sha1-61cc55aa0453184734c3fa0b621eda6fa874bd83",
+        "other": "Dodaj"
+    },
+    "dialog.addOption.title": {
+        "hash": "sha1-05448f4011e4f488f6cef7cfaac5e2364528b304",
+        "other": "Dodaj opcje"
+    },
+    "poll.button.addOption": {
+        "hash": "sha1-05448f4011e4f488f6cef7cfaac5e2364528b304",
+        "other": "Dodaj opcje"
+    },
+    "poll.button.deletePoll": {
+        "hash": "sha1-390ff4fdd0cfef6ef35fbf067a605aa3ea87ecca",
+        "other": "Usuń sondę"
+    },
+    "poll.button.endPoll": {
+        "hash": "sha1-60ddde386bd7190fa3da2ba13f97e3caa625d3e6",
+        "other": "Zakończ sondę"
+    },
+    "poll.endPost.answer.heading": {
+        "few": "{{.Answer}} ({{.Count}} głosów)",
```

Comment:
Are the additional translations for `few` and `many` really needed? I see no difference to `other`.

Reply:
> Thanks for fixing the formatting. Would you also be open on maintaining the polish translation in the future?

sure

> Are the additional translations for few and many really needed? I see no difference to other.

corrected